bin/commands/init/generate/index.ts

```diff
@@ -0,0 +1,73 @@
+/*
+// This program and the accompanying materials are made available
+// under the terms of the Eclipse Public License v2.0 which
+// accompanies this distribution, and is available at
+// https://www.eclipse.org/legal/epl-v20.html
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Copyright Contributors to the Zowe Project.
+*/
+
+import * as std from 'cm_std';
+import * as os from "cm_os";
+import * as xplatform from "xplatform";
+import * as fs from '../../../libs/fs';
+import * as config from '../../../libs/config';
+import * as common from '../../../libs/common';
+import * as zosFs from '../../../libs/zos-fs';
+import * as zosJes from '../../../libs/zos-jes';
+
+export function execute(dryRun?: boolean) {
+  common.requireZoweYaml();
+  const ZOWE_CONFIG=config.getZoweConfig();
+  const tempFile = fs.createTmpFile();
+  zosFs.copyMvsToUss(ZOWE_CONFIG.zowe.setup.dataset.prefix + '.SZWESAMP(ZWEGENER)', tempFile);
```

Comment:
This needs to fix `copyMvsToUss` with `escapeDollar`:
```
copyMvsToUss dataset=ZOWE.PR#3718.$$$$$$$$.@###@.SZWESAMP(ZWEGENER), file=/tmp/zwe-669
cp: FSUM6258 cannot open file "//'ZOWE.PR#3718.50727532507275325072753250727532.@###@.SZWESAMP(ZWEGENER)'": EDC5047I An invalid file name was specified as a function parameter. (errno2=0xC00B028E)
```

Reply:
I will include this change to https://github.com/zowe/zowe-install-packaging/pull/3728